Makefile

```diff
@@ -361,7 +361,7 @@ $(IN_CONTAINER): %-in-container:
 	$(PODMANCMD) run --rm --env HOME=/root \
 		-v $(CURDIR):/src -w /src \
 		--security-opt label=disable \
-		docker.io/library/golang:1.22 \
+		quay.io/libpod/validatepr:latest \
```

Comment:
I tend to disagree with this change from the perspective of the vendor-in-container target. For that using the minimum supported go version is often the best. And with this change it is harder to find out which go version the target is using.
But given the go version was not updated here either when we bumped to go 1.23 I guess that target is not really used anyway for vendoring so I guess I can live with that and for validate that is clearly correct.